src/components/dashboard/Navbar.jsx

```
@@ -17,46 +18,83 @@ const Navbar = ({ handleButtonClick }) => {
-            <button onClick={handleLogout} className="mt-8 px-6 py-3 rounded bg-red-500 hover:bg-red-600 transition text-white text-lg">Cerrar sesión</button>
-        </div>
-    )}
+                    <div className="fixed top-0 left-0 w-screen h-screen bg-[#1e1b4b] text-white flex flex-col items-center justify-center z-40 transition-opacity animate-fade-in">
+                        <h2 className="text-2xl font-bold mb-6">Admin Panel</h2>
+                        {/* <ul className="space-y-6 text-center">
+                            <li><button onClick={handleButtonClick} className="text-lg py-3 px-6 rounded hover:bg-white/10 transition">Dashboard</button></li>
+                            <li><button onClick={handleButtonClick} className="text-lg py-3 px-6 rounded hover:bg-white/10 transition">Users</button></li>
+                            <li><button onClick={handleButtonClick} className="text-lg py-3 px-6 rounded hover:bg-white/10 transition">Settings</button></li>
+                </ul> */}
+                        <button onClick={handleLogout} className="mt-8 px-6 py-3 rounded  hover:bg-red-600 transition text-white text-lg flex items-center gap-2">
+                            <LogOut className="h-5 w-5" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)